data/morrow-base/morrow.yml

```diff
@@ -15,10 +15,20 @@
   components:
   - template
 
+# base entities for all class templates
+- id: morrow:class
+  components:
+  - template
+  - class_definition:
+      health_func: !func '{ level * 10 + 10 }'
```

Comment:
Add documentation to this explaining that this defines the core health curve of all characters.